pl/foundation/annualreport/2012/index.lang

```diff
@@ -95,7 +95,7 @@ Wygenerowanie większości naszych przychodów było możliwe dzięki wbudowanej
 
 
 ;A growing percentage of our revenue also comes from support from the public, including grants and individual donors like you. These donations are directly applied to important projects such as Webmaker, Open Badges and Mozilla-Knight Open News that extend Mozilla's influence in new areas.
-Rosnącym odsetkiem naszych przychodów staje się wsparcie społeczeństwa poprzez dotacje od firm i osób prywatnych, takich jak Ty. Przychody z tego tytułu są kierowane na ważne działania rozszerzające wpływ Mozilli w zupełnie nowych dziedzinach. Wśród takich projektów są między innymi: Webmaker, Open Badges i Mozilla-Knight Open News.
+Rosnącym odsetkiem naszych przychodów staje się wsparcie społeczeństwa poprzez darowizny od firm i osób prywatnych, takich jak Ty. Przychody z tego tytułu są kierowane na ważne działania rozszerzające wpływ Mozilli w zupełnie nowych dziedzinach. Wśród takich projektów są między innymi: Webmaker, Open Badges i Mozilla-Knight Open News.
```

Comment:
I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981